tests/cxx/unit_tests/derivative/nuclear.cpp

```diff
@@ -19,13 +19,17 @@
 #include <simde/energy/ao_energy.hpp>
 #include <vector>
 
-using Gradient = simde::AOEnergyNuclearGradient<std::vector<double>>;
-using Hessian  = simde::AOEnergyNuclearHessian<std::vector<double>>;
+using Gradient   = simde::EnergyNuclearGradientStdVectorD;
+using Hessian    = simde::EnergyNuclearHessianStdVectorD;
+using AOGradient = simde::AOEnergyNuclearGradientStdVectorD;
+using AOHessian  = simde::AOEnergyNuclearHessianStdVectorD;
 
 TEST_CASE("Nuclear Derivative Property Types") {
-    test_property_type<Gradient>({"AOs", "Chemical System", "Arg 1"},
-                                 {"Derivative"});
+    test_property_type<AOGradient>({"AOs", "Chemical System", "Arg 1"},
+                                   {"Derivative"});
 
-    test_property_type<Hessian>({"AOs", "Chemical System", "Arg 1", "Arg 2"},
-                                {"Derivative"});
+    test_property_type<AOHessian>({"AOs", "Chemical System", "Arg 1", "Arg 2"},
+                                  {"Derivative"});
+
+    test_property_type<Gradient>({"Chemical System"}, {"Derivative"});
```

Comment:
@ryanmrichard Shouldn't this have two inputs, e.g. `{"Chemical System", "Arg1"}`?

Reply:
It should, not sure why the tests are passing...